test/integration/app/cold-storage/rebalance-to-cold-wallet.spec.ts

```diff
@@ -0,0 +1,86 @@
+import { ColdStorage } from "@app"
+import * as appConfig from "@config"
+import { btc2sat, toSats } from "@domain/bitcoin"
+import { RebalanceChecker } from "@domain/cold-storage"
+import { BitcoindWalletClient } from "@services/bitcoind"
+import { lndsBalances } from "@services/lnd/utils"
+
+import { bitcoindClient, checkIsBalanced, mineBlockAndSyncAll } from "test/helpers"
+
+jest.mock("@config", () => {
+  const config = jest.requireActual("@config")
+  return {
+    ...config,
+    getColdStorageConfig: jest.fn(() => config.getColdStorageConfig()),
+  }
+})
+
+let coldStorageWalletClient: BitcoindWalletClient
+
+beforeAll(async () => {
+  const { onChainWallet } = appConfig.getColdStorageConfig()
+
+  const wallets = await bitcoindClient.listWallets()
+  const walletName = wallets.find((item) => item.includes(onChainWallet))
+  if (!walletName) throw new Error("Invalid specter wallet name")
+
+  coldStorageWalletClient = new BitcoindWalletClient({ walletName })
```

Comment:
Can probably refactor this class constructor to our single-arg rule if this is in scope

Reply:
Updated https://github.com/GaloyMoney/galoy/pull/986/commits/015a271222bd5297808a18f24b07029d8fab637a